src/main/java/org/jahia/modules/sitemap/services/SitemapService.java

```diff
@@ -47,10 +47,11 @@ public interface SitemapService {
 
     /**
      * Gets sitemap entry cache value for a giving sitemap cache key.
+     * @param siteKey (mandatory)
      * @param key (mandatory)
      * @return sitemap cache content as String.
      */
-    String getSitemap(String key);
+    String getSitemap(String siteKey, String key);
```

Comment:
It would be better to rename that variable to `locale` instead of `key` for better understanding.